### ISSUE TITLE
v0.3.14 error win32 x64

### ISSUE THREAD

**Issue by jordiindahood** [microsoft/vscode-java-installer#290]
:warning:Please attach **%TEMP%\vscode-java-installer\log.txt** and **%TEMP%\vscode-java-installer\AdoptOpenJDK-MSI.log** for more details.

version: 0.3.14
env:
```
JAVA_HOME: undefined
PATH:
C:\Windows\system32
C:\Windows
C:\Windows\System32\Wbem
C:\Windows\System32\WindowsPowerShell\v1.0\
C:\Windows\System32\OpenSSH\
C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR
C:\Program Files\dotnet\
C:\Program Files\Git\cmd
C:\Users\jordy\AppData\Local\Programs\Python\Python310\Scripts\
C:\Users\jordy\AppData\Local\Programs\Python\Python310\
C:\Users\jordy\AppData\Local\Microsoft\WindowsApps
C:\Program Files\JetBrains\PyCharm 2021.2.3\bin

C:\Users\jordy\AppData\Local\Programs\Microsoft VS Code\bin
C:\Users\jordy\.dotnet\tools
C:\MinGW\bin

JDKs:

```
log:
```
[5:45:19 PM] AdoptOpenJDK-11-x64
[5:48:38 PM] Download JDK ... [Completed]
[5:48:38 PM] Install JDK
[5:48:38 PM] Installing AdoptOpenJDK-11-x64...
[5:48:51 PM] Install JDK ... [Completed]
[5:48:51 PM] Configure Java Settings
[5:48:51 PM] Error occurred
[5:48:51 PM] Error: Java Home not found.

```

**Comment by jordiindahood** [comment]
log: 
info: Started {"timestamp":"2021-11-28T16:45:16.453Z"}
verbose: Ids: {"uiKind":1,"appRoot":"C:\\Users\\jordy\\AppData\\Local\\Temp\\1u7C39sHofooqVi4tC6fT2m72NU\\resources\\app.asar","isTelemetryEnabled":true,"machineId":"b290c634fccfd340f1dbb04c57a18ee374b2aa1c48a19569e8e2a22ae7cb62b6","sessionId":"34a03a5e-ab39-48e4-9638-ff9c504e8fb21638117916055","timestamp":"2021-11-28T16:45:16.454Z"}
verbose: Packed binaries: {"timestamp":"2021-11-28T16:45:18.037Z"}
verbose: Found JDKs in env.JAVA_HOME {"jdks":[],"timestamp":"2021-11-28T16:45:18.038Z"}
verbose: Found JDKs in env.PATH {"jdks":[],"timestamp":"2021-11-28T16:45:18.050Z"}
verbose: Found JDKs in WindowsRegistry {"jdks":[],"timestamp":"2021-11-28T16:45:18.118Z"}
verbose: Found JDKs in DefaultLocation {"jdks":[],"timestamp":"2021-11-28T16:45:18.121Z"}
verbose: Looking for Code stable x64-user in x64 registry: HKCU\Software\Microsoft\Windows\CurrentVersion\Uninstall\{771FD6B0-FA20-440A-A002-3B3BAC16DC50}_is1 {"timestamp":"2021-11-28T16:45:18.122Z"}
verbose: Code stable x64-user is installed. {"timestamp":"2021-11-28T16:45:18.174Z"}
verbose: Installed components: {"vscode":{"homedir":"C:\\Users\\jordy\\AppData\\Local\\Programs\\Microsoft VS Code\\","version":"1.62.3"},"extensions":[],"timestamp":"2021-11-28T16:45:18.174Z"}
verbose: jdk metadata {"release_name":"jdk-11.0.13+8","release_link":"https://github.com/adoptium/temurin11-binaries/releases/tag/jdk-11.0.13%2B8","timestamp":"2021-11-15T16:03:02Z","release":true,"binaries":[{"os":"windows","architecture":"x64","binary_type":"jdk","openjdk_impl":"hotspot","binary_name":"OpenJDK11U-jdk_x64_windows_hotspot_11.0.13_8.zip","binary_link":"https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.13%2B8/OpenJDK11U-jdk_x64_windows_hotspot_11.0.13_8.zip","binary_size":196666656,"checksum_link":"https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.13%2B8/OpenJDK11U-jdk_x64_windows_hotspot_11.0.13_8.zip.sha256.txt","installer_name":"OpenJDK11U-jdk_x64_windows_hotspot_11.0.13_8.msi","installer_link":"https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.13%2B8/OpenJDK11U-jdk_x64_windows_hotspot_11.0.13_8.msi","installer_size":175078400,"installer_checksum_link":"https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.13%2B8/OpenJDK11U-jdk_x64_windows_hotspot_11.0.13_8.msi.sha256.txt","installer_download_count":197344,"version":"11","version_data":{"openjdk_version":"11.0.13+8","semver":"11.0.13+8"},"heap_size":"normal","download_count":222391,"updated_at":"2021-10-24T09:54:24Z"}],"download_count":555574}
info: JDK downloaded. {"timestamp":"2021-11-28T16:48:38.833Z"}
verbose: Installing JDK with command: {"msiexec":"C:\\Windows\\System32\\msiexec.exe","args":["/i","\"C:\\Users\\jordy\\AppData\\Local\\Temp\\vscode-java-installer\\binaries\\jdk\\11\\x64\\OpenJDK11U-jdk_x64_windows_hotspot_11.0.13_8.msi\"","MSIINSTALLPERUSER=1","ADDLOCAL=FeatureJavaHome,FeatureEnvironment,FeatureJarFileRunWith","/passive","/l*v \"C:\\Users\\jordy\\AppData\\Local\\Temp\\vscode-java-installer\\AdoptOpenJDK-MSI.log\""],"timestamp":"2021-11-28T16:48:38.842Z"}
verbose: Found JDKs in env.JAVA_HOME {"jdks":[],"timestamp":"2021-11-28T16:48:51.871Z"}
verbose: Found JDKs in env.PATH {"jdks":[],"timestamp":"2021-11-28T16:48:51.884Z"}
verbose: Found JDKs in WindowsRegistry {"jdks":[],"timestamp":"2021-11-28T16:48:51.974Z"}
verbose: Found JDKs in DefaultLocation {"jdks":[],"timestamp":"2021-11-28T16:48:51.974Z"}
error: Java Home not found. {"timestamp":"2021-11-28T16:48:51.975Z"}
verbose: Found JDKs in env.JAVA_HOME {"jdks":[],"timestamp":"2021-11-28T17:01:54.367Z"}
verbose: Found JDKs in env.PATH {"jdks":[],"timestamp":"2021-11-28T17:01:54.384Z"}
verbose: Found JDKs in WindowsRegistry {"jdks":[],"timestamp":"2021-11-28T17:01:54.479Z"}
verbose: Found JDKs in DefaultLocation {"jdks":[],"timestamp":"2021-11-28T17:01:54.480Z"}

**Comment by jordiindahood** [comment]
### info: Started {"timestamp":"2021-11-28T16:45:16.453Z"}
verbose: Ids: {"uiKind":1,"appRoot":"C:\\Users\\jordy\\AppData\\Local\\Temp\\1u7C39sHofooqVi4tC6fT2m72NU\\resources\\app.asar","isTelemetryEnabled":true,"machineId":"b290c634fccfd340f1dbb04c57a18ee374b2aa1c48a19569e8e2a22ae7cb62b6","sessionId":"34a03a5e-ab39-48e4-9638-ff9c504e8fb21638117916055","timestamp":"2021-11-28T16:45:16.454Z"}
verbose: Packed binaries: {"timestamp":"2021-11-28T16:45:18.037Z"}
verbose: Found JDKs in env.JAVA_HOME {"jdks":[],"timestamp":"2021-11-28T16:45:18.038Z"}
verbose: Found JDKs in env.PATH {"jdks":[],"timestamp":"2021-11-28T16:45:18.050Z"}
verbose: Found JDKs in WindowsRegistry {"jdks":[],"timestamp":"2021-11-28T16:45:18.118Z"}
verbose: Found JDKs in DefaultLocation {"jdks":[],"timestamp":"2021-11-28T16:45:18.121Z"}
verbose: Looking for Code stable x64-user in x64 registry: HKCU\Software\Microsoft\Windows\CurrentVersion\Uninstall\{771FD6B0-FA20-440A-A002-3B3BAC16DC50}_is1 {"timestamp":"2021-11-28T16:45:18.122Z"}
verbose: Code stable x64-user is installed. {"timestamp":"2021-11-28T16:45:18.174Z"}
verbose: Installed components: {"vscode":{"homedir":"C:\\Users\\jordy\\AppData\\Local\\Programs\\Microsoft VS Code\\","version":"1.62.3"},"extensions":[],"timestamp":"2021-11-28T16:45:18.174Z"}
verbose: jdk metadata {"release_name":"jdk-11.0.13+8","release_link":"https://github.com/adoptium/temurin11-binaries/releases/tag/jdk-11.0.13%2B8","timestamp":"2021-11-15T16:03:02Z","release":true,"binaries":[{"os":"windows","architecture":"x64","binary_type":"jdk","openjdk_impl":"hotspot","binary_name":"OpenJDK11U-jdk_x64_windows_hotspot_11.0.13_8.zip","binary_link":"https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.13%2B8/OpenJDK11U-jdk_x64_windows_hotspot_11.0.13_8.zip","binary_size":196666656,"checksum_link":"https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.13%2B8/OpenJDK11U-jdk_x64_windows_hotspot_11.0.13_8.zip.sha256.txt","installer_name":"OpenJDK11U-jdk_x64_windows_hotspot_11.0.13_8.msi","installer_link":"https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.13%2B8/OpenJDK11U-jdk_x64_windows_hotspot_11.0.13_8.msi","installer_size":175078400,"installer_checksum_link":"https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.13%2B8/OpenJDK11U-jdk_x64_windows_hotspot_11.0.13_8.msi.sha256.txt","installer_download_count":197344,"version":"11","version_data":{"openjdk_version":"11.0.13+8","semver":"11.0.13+8"},"heap_size":"normal","download_count":222391,"updated_at":"2021-10-24T09:54:24Z"}],"download_count":555574}
info: JDK downloaded. {"timestamp":"2021-11-28T16:48:38.833Z"}
verbose: Installing JDK with command: {"msiexec":"C:\\Windows\\System32\\msiexec.exe","args":["/i","\"C:\\Users\\jordy\\AppData\\Local\\Temp\\vscode-java-installer\\binaries\\jdk\\11\\x64\\OpenJDK11U-jdk_x64_windows_hotspot_11.0.13_8.msi\"","MSIINSTALLPERUSER=1","ADDLOCAL=FeatureJavaHome,FeatureEnvironment,FeatureJarFileRunWith","/passive","/l*v \"C:\\Users\\jordy\\AppData\\Local\\Temp\\vscode-java-installer\\AdoptOpenJDK-MSI.log\""],"timestamp":"2021-11-28T16:48:38.842Z"}
verbose: Found JDKs in env.JAVA_HOME {"jdks":[],"timestamp":"2021-11-28T16:48:51.871Z"}
verbose: Found JDKs in env.PATH {"jdks":[],"timestamp":"2021-11-28T16:48:51.884Z"}
verbose: Found JDKs in WindowsRegistry {"jdks":[],"timestamp":"2021-11-28T16:48:51.974Z"}
verbose: Found JDKs in DefaultLocation {"jdks":[],"timestamp":"2021-11-28T16:48:51.974Z"}
error: Java Home not found. {"timestamp":"2021-11-28T16:48:51.975Z"}
verbose: Found JDKs in env.JAVA_HOME {"jdks":[],"timestamp":"2021-11-28T17:01:54.367Z"}
verbose: Found JDKs in env.PATH {"jdks":[],"timestamp":"2021-11-28T17:01:54.384Z"}
verbose: Found JDKs in WindowsRegistry {"jdks":[],"timestamp":"2021-11-28T17:01:54.479Z"}
verbose: Found JDKs in DefaultLocation {"jdks":[],"timestamp":"2021-11-28T17:01:54.480Z"}

**Comment by jordiindahood** [comment]
[AdoptOpenJDK-MSI.log](https://github.com/microsoft/vscode-java-installer/files/7613972/AdoptOpenJDK-MSI.log)
[log.txt](https://github.com/microsoft/vscode-java-installer/files/7613974/log.txt)

**Comment by Eskibear** [comment]
this bug should be fixed in latest version v0.3.17, download here https://aka.ms/vscode-java-installer-win  

Let me know if it works.

**Comment by jordiindahood** [comment]
> this bug should be fixed in latest version v0.3.17, download here https://aka.ms/vscode-java-installer-win
> 
> Let me know if it works.

it wokrked i guess 
thank you for your help
